core/Plugin/Dependency.php

```diff
@@ -77,6 +77,25 @@ public function setPiwikVersion($piwikVersion)
         $this->piwikVersion = $piwikVersion;
     }
 
+    /**
+     * Returns a string describing the missing requirements for this plugin
+     *
+     * @param array $requires
+     * @return string "PIWIK >=3.0.0, PHP >= 5.5"
+     */
+    public function getMissingDependenciesAsString($requires)
+    {
+        $missingDependencies = $this->getMissingDependencies($requires);
+
+        $causedBy = array();
+        foreach ($missingDependencies as $dependency) {
+            $causedBy[] = strtoupper($dependency['requirement']) . ' ' . $dependency['causedBy'];
```

Comment:
I think this should be rather capitalize instead of strtoupper. There will be also plugin names etc. For PHP we have in the Marketplace a special if to convert it to upper case